<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MissionWidget</class>
 <widget class="QWidget" name="MissionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="editName">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="taskTabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="taskPropertiesTab">
      <attribute name="title">
       <string>Properties</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>98</width>
            <height>28</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="loadDefaultButton">
            <property name="text">
             <string>Default Values</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonReset">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Undo Changes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonSubmit">
            <property name="text">
             <string>Submit</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPayload">
      <attribute name="title">
       <string>Payload</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QListView" name="payloadList">
         <property name="spacing">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>At Start:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QToolButton" name="buttonPayloadEnable">
              <property name="toolTip">
               <string>Enable the payload at begin</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../resources.qrc">
                <normaloff>:/:/plugins/PathPlanner_3/icons/stateEnable.png</normaloff>:/:/plugins/PathPlanner_3/icons/stateEnable.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="buttonPayloadDisable">
              <property name="toolTip">
               <string>Disable the payload at begin</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../resources.qrc">
                <normaloff>:/:/plugins/PathPlanner_3/icons/stateDisable.png</normaloff>:/:/plugins/PathPlanner_3/icons/stateDisable.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="buttonPayloadKeepState">
              <property name="toolTip">
               <string>Clear settings at begin</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../resources.qrc">
                <normaloff>:/:/plugins/PathPlanner_3/icons/stateKeep.png</normaloff>:/:/plugins/PathPlanner_3/icons/stateKeep.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>At End:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QToolButton" name="buttonPayloadEnableAtEnd">
              <property name="toolTip">
               <string>Enable the payload at end</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../resources.qrc">
                <normaloff>:/:/plugins/PathPlanner_3/icons/stateEnable.png</normaloff>:/:/plugins/PathPlanner_3/icons/stateEnable.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="buttonPayloadDisableAtEnd">
              <property name="toolTip">
               <string>Disable the payload at end</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../resources.qrc">
                <normaloff>:/:/plugins/PathPlanner_3/icons/stateDisable.png</normaloff>:/:/plugins/PathPlanner_3/icons/stateDisable.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="buttonPayloadClearStateAtEnd">
              <property name="toolTip">
               <string>Clear settings at end</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../resources.qrc">
                <normaloff>:/:/plugins/PathPlanner_3/icons/stateKeep.png</normaloff>:/:/plugins/PathPlanner_3/icons/stateKeep.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
